### DISCUSSION TITLE
Kill Count, veterans and more Forts

### DISCUSSION THREAD

**Discussion by Purakane** [Keriew/augustus#462] [Ideas]
Hey guys,

It would be great to see a kill count for each legion and the ability to have veteran legions once they reach a certain kill count. 

Please bring back more forts like 10 to facilitate fewer but larger battles as the small attacks/frequent attacks are more of a nuisance than a threat. The battles are unchallenging even on very hard mode and would find myself intentionally triggering Caesar's attacks by lowering my favor to add more spice to the game.

**Comment by Edu2014** [comment]
you can increase the amount of forts if you make a hades sanctuary. It just takes too long to make anything that needs clay, wood and marble. I even asked if they can't speed it up.

**Reply by Purakane** [reply]
Yes, the monument system is still glitchy in the sense that when all requires are met the builders still will not bring materials to the site. I've tested this and I believe one of the factors may be that they cannot take resources from a warehouse that is set on "getting."

**Reply by Purakane** [reply]
Also I'm unable to find the hades sanctuary .

**Reply by thomasherena** [reply]
How do you build a Hades sanctuary??
